src/DataSource/DB.hs

```diff
@@ -46,6 +47,17 @@ cachedZendeskLayer = ZendeskLayer
 -- Database instances
 -- FROM
 
+instance FromField TicketFieldId where
+    fromField (Field (SQLInteger tfId) _)     = Ok . TicketFieldId . fromIntegral $ tfId
+    fromField f                             = returnError ConversionFailed f "need a text, ticket status"
+
+instance FromField TicketFieldValue where
+    fromField (Field (SQLText tfValue) _)     = Ok . TicketFieldValue $ tfValue
+    fromField f                             = returnError ConversionFailed f "need a text, ticket status"
+
+instance FromField [TicketField] where
+    fromField _ = undefined
```

Comment:
?? There should be a warning, try fixing this.

Reply:
Yes I will! Thank you!